src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <ul>
+    {movies.map(movie => (
+      <li key={movie.imdbId}>
+        <MovieCard {...movie} />
+      </li>
```

Comment:
```suggestion
      <MovieCard {...movie} />
```

Тег `<li>` краще винести в компоненту MovieCard